pkg/controller/rbac_jobs.go

```diff
@@ -519,3 +529,51 @@ func (c *StashController) ensureBackupJobRoleBinding(resource *core.ObjectRefere
 	})
 	return err
 }
+
+func (c *StashController) checkAndGetWorrkloadReference(bcObj *api_v1beta1.BackupConfiguration) (bool bool, ref *core.ObjectReference) {
```

Comment:
You also have to handle
- DeploymentConfig
- PersistentVolumeClaim
- AppBinding